index.js

```diff
@@ -16,43 +52,57 @@ module.exports = {
       name: options.name,
 
       defaultConfig: {
-        configuredServices: {
-          bugsnag: {
-            url: 'http://notify.bugsnag.com/deploy'
+        configuredServices: function(context) {
+          return {
+            bugsnag: {
+              url: 'http://notify.bugsnag.com/deploy',
+              method: 'POST',
+              headers: {},
+              body: function() {
+                var apiKey = this.apiKey;
+
+                if (!apiKey) { return; }
```

Comment:
Because of the guard on line 37 we can just return a falsy value for `url`, 'headers', 'method' or 'body' and the plugin won't notify the service. I'm not too stoked about the api here. So maybe we could mark configuration options as 'required' to make this more elegant.